server/routerlicious/packages/tinylicious/package.json

```diff
@@ -104,7 +104,6 @@
 		"ts-node": "^8.6.2",
 		"typescript": "~4.5.5"
 	},
-	"packageManager": "pnpm@7.33.5+sha512.4e499f22fffe5845aa8f9463e1386b2d72c3134e0ebef9409360ad844fef0290e82b479b32eb2ec0f30e56607e1820c22167829fd62656d41a8fd0cc4a0f4267",
```

Comment:
This should have been removed when this package was moved into the server release group.